software/spmd/vcache_atomics/main.c

```diff
@@ -0,0 +1,57 @@
+#include "bsg_manycore.h"
```

Comment:
is this test meant to be run in single threaded execution?

Reply:
It's not a problem if so, I'm just asking to know.

Reply:
Yes